pallas-configs/src/shelley.rs

```diff
@@ -154,6 +233,25 @@ mod tests {
         from_file(&path).unwrap()
     }
 
+    #[test]
+    fn calc_address_txid() {
+        let config = load_test_data_config("golden");
+        let utxos = shelley_utxos(&config);
+        let utxo = utxos.first().unwrap();
+        assert_eq!(
+            utxo.0.to_string(),
+            "f9ec23569778d1c5f7f43e0e98464335f02fb98b57683faa1c6b18c82921d2da"
+        );
+        assert_eq!(
+            utxo.1.to_bech32().unwrap(),
+            "addr_test1qrsm4h32h9r95f8at64ykuugxqu3wvu0s5ay3vg6tlyevjh4e2flkegka00r69gt8c4vkxgf2vnnph3nsvhlkg5ukgxslee3tf"
+        );
+        assert_eq!(
+            utxo.2,
+            12157196
+        );
+    }
```

Comment:
I also tested this gives the same UTXO as running a yaci-devkit local node with this genesis block 👍